trin-history/src/lib.rs

```diff
@@ -113,3 +116,16 @@ pub fn spawn_history_network(
             .expect("failed to pause until ctrl-c");
     })
 }
+
+pub fn spawn_history_heartbeat(network: Arc<HistoryNetwork>) {
+    tokio::spawn(async move {
+        let mut heart_interval = interval(Duration::from_millis(30000));
+
+        loop {
+            let radius = network.overlay.data_radius();
+            let coverage_percent = *radius / U256::MAX * 100;
+            info!("status: coverage={coverage_percent:.2}%");
```

Comment:
At some point, I'd expect we'd want these heartbeat logs to be their own "level" - so a user can run them at just that level, hiding the rest of the `info!` logs.

For now, it might be worth something slightly more descriptive like `status: history network coverage=..` ... or maybe even `heartbeat: history network coverage=..` - so that we can distinguish the heartbeat logs from regular `info!` logs.

Reply:
Thinking a bit more about this, just wanted to mention that it's probably worth moving towards a structure where the heartbeat log is in some way connected to our metrics - or at the very least they share the data which they report. Since something like this `coverage_percent` (as well as most of the heartbeat log data I'd imagine) would be nice to have displayed on the metrics dashboard